src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
```

Comment:
PullProofAI

Potential performance issue with the length of `organizationName` being set to 255. Consider reducing the length if it exceeds the actual maximum length required for organization names to optimize storage and retrieval operations.

```diff
- @Column({ type: 'varchar', length: 255 })
+ @Column({ type: 'varchar', length: 50 }) // Adjust the length based on actual requirements
```


<!-- This is an auto-generated comment by PullProofAI -->